<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QDialog" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>-1</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(0, 120, 80, 255));</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Carlito,sans-serif'; font-size:14pt; font-weight:600; color:#323232;&quot;&gt;School Management System&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>191</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="StudentsBtn">
      <property name="text">
       <string>Students</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StaffBtn">
      <property name="text">
       <string>Staff</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClassBtn">
      <property name="text">
       <string>Class</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MainMenuBtn">
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
